packages/amplify-frontend-android/lib/frontend-config-creator.js

```diff
@@ -43,6 +43,9 @@ function getAWSConfigObject(amplifyResources) {
     },
   };
 
+  if (amplifyResources.testMode) {
+    configOutput.DangerouslyConnectToHTTPEndpointForTesting = true;
```

Comment:
Is this needed? We decided not to have a global flag

Reply:
yes, removed it